data/features/featureList.yaml

```diff
@@ -0,0 +1,891 @@
+## Annotated examples
+## Just for documentation, not used by code. Please leave intact.
+examples:
+
+  # A feature gate that is alpha, enabled, and still configurable
+- name: ExampleFeature1
+  alpha:
+    fromVersion: v1.3
+    state: true # override the default state for this stage (false)
+  deprecated:
+    fromVersion: v1.18
+
+  # An experiment that didn't work out. After a while in alpha, it
+  # was deprecated and can no longer be enabled.
+- name: ExampleFeature2
+  alpha:
+    fromVersion: v1.15
+  deprecated:
+    fromVersion: v1.18
+
+  # A feature gate that is no longer available. This started as a
+  # beta featured but, unusually, one that was initially disabled
+  # by default.
+  # Later, it was enabled by default (still beta), eventually
+  # graduating to general availability.
+- name: ExampleFeature3
+  beta:
+    fromVersion: v1.4
+    state: false #
+    enabledFromVersion: v1.5
+  stable:
+    fromVersion: v1.7
+
+featureGateList:
+# Sorted alphabetically
+- name: Accelerators
+  alpha:
+    fromVersion: v1.6
+  deprecated:
+    fromVersion: v1.11
+- name: AdvancedAuditing
+  alpha:
+    fromVersion: v1.7
+  beta:
+    fromVersion: v1.8
+  stable:
+    fromVersion: v1.12
+- name: AffinityInAnnotations
+  alpha:
+    fromVersion: v1.6
+  deprecated:
+    fromVersion: v1.8
+- name: AllowInsecureBackendProxy
+  beta:
+    fromVersion: v1.17
+- name: AllowExtTrafficLocalEndpoints
+  beta:
+    fromVersion: v1.4
+    state: false
+  stable:
+    fromVersion: v1.7
+- name: AnyVolumeDataSource
+  alpha:
+    fromVersion: v1.18
+- name: APIListChunking
+  alpha:
+    fromVersion: v1.8
+  beta:
+    fromVersion: v1.9
+- name: APIPriorityAndFairness
+  alpha:
+    fromVersion: v1.17
+  beta:
+    fromVersion: v1.20
+- name: APIResponseCompression
```

Comment:
Snippet from Alpha/Beta Feature Gate table
```
| `APIPriorityAndFairness` | `false` | Alpha | 1.17 | 1.19 |
| `APIPriorityAndFairness` | `true` | Beta | 1.20 | |
```

Reply:
I don't see anything to fix - is there a bug here?

Reply:
Nothing to fix. Showing what the data looks like in Markdown.